python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -669,7 +673,7 @@ class RandomForestRegressor(Base):
                                              storage_type=storage_type)
 
         preds = tl_to_fil_model.predict(X, out_type)
```

Comment:
why do we not use `self._obtain_treelite_handle()` function in place of lines 654-661 in gpu predict?

Reply:
added now